src/components/task7-8/counter.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export default class Counter extends React.Component {
+    constructor(props){
+        super(props);
+
+        this.state = {
+            score: 0,
+        }
+    }
+    
+    incScore = () => {
+        if (this.state.score === 10)
+            this.setState({score: 0})
+        else      
+            this.setState(prevState => {
+                return {score: prevState.score + 1}
```

Comment:
Even though this will work, this code can lead to unnecessary bugs. Always use `{}` in your if else statements